Gemfile

```diff
@@ -10,6 +10,12 @@ gemspec
 # checkout the latest version (develop) from github.
 allow_local = ENV['FAVOR_LOCAL_GEMS']
 
+# uncomment when you want CI to use develop branch of extension gem
+gem 'openstudio-extension', github: 'NREL/OpenStudio-extension-gem', branch: 'develop'
+
```

Comment:
Do you want to track a branch here?  There is risk of things breaking and becoming out of sync with changes. I usually like to do this for testing only and then change to either a git tag or use the official rubygems version if available.

Reply:
@tijcolem I'm fine having this be the same version of gem in the 3.2 installer.